packages/lib-classifier/src/plugins/drawingTools/components/Circle/Circle.spec.js

```diff
@@ -1,27 +1,55 @@
 import React from 'react'
-import { render, screen } from '@testing-library/react'
+import { shallow, mount } from 'enzyme'
```

Comment:
I think reverting to Enzyme from React Testing Library might be a bad idea. Enzyme has held us up from updating React in the past, and 18 is a release candidate now.

Reply:
I agree. My thought was to quick get a test out there so this is at least covered with a test for now, then come back to it at some point to convert to RTL... but we all know how that goes! ;) So, I will take the time to dive into RTL more and figure out how to convert this now.